.github/workflows/deploy.yml

```
@@ -0,0 +1,99 @@
+            echo "✅ Frontend Deployment Done!"
+
+      # ✅ 성공 알림
+      - name: Send deployment status to Discord
+        if: success()
+        uses: tsickert/discord-webhook@v5.3.0
+        with:
+          webhook-url: ${{ secrets.DISCORD_WEBHOOK_URL }}
+          content: |
+            [OPEN-ALOC-FRONTEND]
+            ✅ 배포 성공
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)